mllib-dal/src/main/native/PCADALImpl.cpp

```diff
@@ -32,6 +32,136 @@ using namespace daal::services;
 
 typedef double algorithmFPType; /* Algorithm floating-point type */
 
+static void pca_compute(JNIEnv *env, jobject obj, int rankId,
```

Comment:
Could you still use the name doPCADALCompute and remove the original function since it's no longer used.